src/components/widgets/registration/useGetReviewInfo.tsx

```diff
@@ -95,7 +106,7 @@ export const useGetReviewInfo = (
                         {extractDate(date)}
                     </Typography.Title>
                     <Typography.Paragraph>
-                        {JSON.stringify(selectedTimeSlots)}
+                        {mergedSelectedTimeSlotsLabels.join(` ${t('and')} `)}
```

Comment:
I guess this way it will look better 
<img width="496" alt="image" src="https://github.com/ledenev-viktor/music-studio/assets/48691234/36af28ec-dbb6-464a-8d4e-f59c4b20043c">